lms/resources/_js_config.py

```diff
@@ -257,3 +304,10 @@ def _hypothesis_client(self):
                 }
             ]
         }
+
+    def _is_canvas(self):
+        """Return True if Canvas is the LMS that launched us."""
+        return (
+            self._request.params.get("tool_consumer_info_product_family_code")
+            == "canvas"
+        )
```

Comment:
I don't think this is needed as it looks like a dupe of `context.is_canvas()`

Reply:
Yup